### DISCUSSION TITLE
Welcome to CodeAssist Discussions!

### DISCUSSION THREAD

**Discussion by tyron12233** [tyron12233/CodeAssist#165] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Amir-org4** [comment]
There is a problem.  Problems such as: We can not select the project storage folder.  The preview layout page is broken. When setting the buttons or other settings of the application page, the preview page is not visible and only a small part of it is visible. Lack of support for Persian language is another problem of this program.

**Reply by steven750MC** [reply]
صلم دختری؟

**Comment by Amir-org4** [comment]
The settings page for editing, the page of the program we want to design is broken and will not be displayed.

**Comment by PARSOJ1** [comment]
how to add image or file in drawable(android 11)?

**Comment by FlutterGenerator** [comment]
Plz update 2025

**Comment by steven750MC** [comment]
We need compose material3, you are Best!

**Comment by malakasambo37-creator** [comment]
Please add a build output feature so that when you create an application you don't need to build it first so you can see the results directly, like when you create a website.
